WordPress/Classes/Utility/WPMapFilterReduce.m

```diff
@@ -11,7 +11,7 @@ - (instancetype)wp_map:(WPMapBlock)mapBlock
             [results addObject:objectToAdd];
         }
     }
-    return [[self class] arrayWithArray:results];
+    return [NSArray arrayWithArray:results];
```

Comment:
We don't, but now that you mention it, the return type should be `NSArray` and not `instancetype` so good catch :smile: 

If you look at the commit message in ffa729ea976961714fb379c106e4c57378c143e7, the previous initializer wasn't working:

> Prevents a crash when calling the methods. Since `[self class]` could be
`__NSCFArray` or another internal class, the constructor would throw an
exception:

>  *** -[NSMutableArray initWithCapacity:]: method only defined for abstract class.  Define -[__NSCFArray initWithCapacity:]!

> Let's be explicit about what we want